src/subToSimFcn.m

```diff
@@ -0,0 +1,186 @@
+function subToSimFcn(subsystem, simulinkFcnName, visibility)
+% subToSimFunc              Converts a subsystem to a Simulink Function
+%
+% Inputs:
+%   subsystem               Path of a subsystem to be converted
+%   simulinkFcnName         Name of the Simulink Function to be created
+%   visibility              Set function visibility to'scoped' or 'global'
+%
+% Outputs:
+%   N/A
+%
+% Example:
+%   subToSimFunc('Demo_Example/f_Sensor_Trip_1', 'f_Sensor_Trip_i', 'scoped')
+%
+%           Converts 'f_Sensor_Trip_1' subsystem to a
+%           scoped Simulink-function 'f_SensorTrip_i'
+
+    %% Input Validation
+    
+    % Check that the subsystem is loaded
+    try
+        assert(ischar(subsystem));
+        assert(bdIsLoaded(bdroot(subsystem)));
+    catch
+        error('Invalid subsystem. Model may not be loaded or name is invalid.');
+    end
+    
+    % Check that the function name is valid
+    try
+        assert(isvarname(simulinkFcnName));
+    catch
+        error('Invalid function name. Use a valid MATLAB variable name.');
+    end
+    
+    % Check that the function visibility is valid
+    try
+        assert(strcmp(visibility, 'scoped') || strcmp(visibility, 'global'));
+    catch
+        error('Invalid function visibility. Use scoped/global visibility.');
+    end
+    
+    %% Add Trigger to Subsystem
+    
+    % Disable subsystem link
+    set_param(subsystem, 'LinkStatus', 'inactive');
```

Comment:
Can you check if the `LinkStatus` is okay to leave as 'inactive', or do we want to break it entirely by setting it to 'none'? See this for more info: https://www.mathworks.com/help/simulink/ug/control-linked-block-status-programmatically.html